lib/galaxy/util/oset.py

```diff
@@ -2,61 +2,70 @@
 Ordered set implementation from https://code.activestate.com/recipes/576694/
 """
 from collections.abc import MutableSet
+from typing import 
+from typing import (
+    Any,
+    Dict,
+    List,
+    TYPE_CHECKING,
+)
 
+if TYPE_CHECKING:
+    from bioblend.galaxy import GalaxyInstance
 
 class OrderedSet(MutableSet):
-    def __init__(self, iterable=None):
+    def __init__(self, iterable=None) -> List[Dict[str, Any]]:
         self.end = end = []
         end += [None, end, end]  # sentinel node for doubly linked list
         self.map = {}  # key --> [key, prev, next]
         if iterable is not None:
             self |= iterable
 
-    def __len__(self):
+    def __len__(self) -> Dict[str, Any]:
         return len(self.map)
 
-    def __contains__(self, key):
+    def __contains__(self, key) -> Dict[str, Any]:
         return key in self.map
 
-    def add(self, key):
+    def add(self, key) -> Dict[str, Any]:
```

Comment:
that's a None. 
Same goes for the rest of the methods: check what the method returns - that will be its return type for the type hint.